mapie/regression/regression.py

```diff
@@ -500,14 +500,22 @@ def fit(
             train/test set.
             By default ``None``.
 
-        **fit_params : dict
-            Additional fit parameters.
+        kwargs : dict
+            Additional fit and predict parameters.
 
         Returns
         -------
         MapieRegressor
             The model itself.
         """
+        fit_params = kwargs.pop('fit_params', {})
+        predict_params = kwargs.pop('predict_params', {})
+
+        if len(predict_params) > 0:
+            self._predict_params = True
```

Comment:
Why are we calling it `_predict_params`?

Reply:
Because scikit-learn only allows the addition of 'private' attributes in its model instances when these attributes are defined by a model method, in this case fit method

I encountered an error on the unit tests checking compliance with scikit-learn standards